light_curves/lc_classifier.ipynb

```diff
@@ -866,6 +918,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%time\n",
```

Comment:
These two classifier cells give me `ValueError`s complaining about missing values, but I assume that's part of what's still being worked on in this notebook.

Reply:
Does the output of `check_is_mtype` all look ok?  I don't get errors on these cells, but yes, I do plan on working on this section more.

Reply:
I'm not sure what it's supposed to look like, but here's what I get:

```
(True,
 None,
 {'is_univariate': False,
  'is_empty': False,
  'has_nans': True,
  'n_instances': 325,
  'is_one_series': False,
  'is_equal_length': True,
  'is_equally_spaced': True,
  'n_panels': 1,
  'is_one_panel': True,
  'mtype': 'pd-multiindex',
  'scitype': 'Panel'})
```